backend/src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+
+/** A user with saved preferences. */
+public final class User {
+
+  /** Datastore ID */
```

Comment:
Add a TODO + issue to add user interests to this model as well?

Reply:
Added TODO and created an issue.